src/tests/networkcontrol/NetworkControl_Status_003.test.js

```diff
@@ -0,0 +1,40 @@
+import {
+  pluginDeactivate,
+  pluginActivate,
+  setNetworkStatus,
+} from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+
+export default {
+  title: 'NetworkControl - Up Status 003',
+  description: 'Set the Network Up status to True',
+  steps: [
+    {
+      description: 'Check if NetworkControl Plugin is stopped correctly',
+      test: pluginDeactivate,
+      params: constants.networkControlPlugin,
+      assert: 'deactivated',
+    },
+    {
+      description: 'Check if NetworkControl Plugin is started correctly',
+      test: pluginActivate,
+      params: constants.networkControlPlugin,
+      assert: 'activated',
+    },
+    {
+      description: 'Invoke Up to check network status',
+      sleep: 5,
+      test() {
+        return setNetworkStatus.call(this, 'eth0', 'false') //TODO - Network status cannot be set to false as the connection is lost.
```

Comment:
We could support this if we add user interaction?

E.g. we could:
- Warn the user his box is going down after running this test
- Execute eth down
- Test if the box is really down (fail if its still online)
- Prompt user to reboot it

wdyt @michielvandergeest @Divya-563 ?